src/remotebuild/test/server.ts

```diff
@@ -122,7 +122,7 @@ describe("server", function (): void {
             fs.existsSync(path.join(certsDir, "ca-key.pem")).should.be.ok;
             fs.existsSync(path.join(certsDir, "ca-cert.pem")).should.be.ok;
         }).then(function (): Q.Promise<{}> {
-            var deferred = Q.defer();
+            var deferred: Q.Deferred<any> = Q.defer();
             // Disabling strict SSL for unit testing
             request.get({ url: "https://localhost:3000", strictSSL: false }, function (error: any, response: any, body: any): void {
```

Comment:
According to this file: https://github.com/borisyankov/DefinitelyTyped/blob/master/request/request.d.ts it seems that response is a http.IncomingMessage